Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = props =>{
+    return(
+        <div>
+            <strong> {props.name} </strong>
```

Comment:
The use of props is great, however, best practices: you can hard-code the values you referenced in HeaderContent here in HeaderTitle: ```return( <div> <strong> Lambda School </strong>``` etc. Makes the code easier to read.